Jenkinsfile

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-@Library('jenkinsfile-lib')_
+@Library('jenkinsfile-lib@PAL-929-dockerfile-lightweight-images')_
```

Comment:
Need to revert this before Merging, but after merging the Jenkinslib

Reply:
done